index.html

```diff
@@ -45,6 +45,55 @@
     <!-- end of NumbersCounter -->
 
     <!-- LatestFeaturedProjects -->
+    <section class="container">
+      <div class="row">
+        <div class="col-10 ml-1 col-md-8 ml-md-2">
```

Comment:
pagal dizaina sakyciau kad cia col-12. tik kad tekstas netelpa tai del to persikelia